auv_control.orogen

```diff
@@ -6,6 +6,7 @@ import_types_from "6dControl.hpp"
 using_library "motor_controller"
 
 typekit.export_types '/motor_controller/PIDSettings'
+typekit.export_types '/base/LinearAngular6DCommand'
```

Comment:
This shouldn't be necessary since 'base' is already exporting the type and 'base' is imported in here.